src/components/PostInfo/PostInfo.tsx

```diff
@@ -1,5 +1,48 @@
 import React from 'react';
+import './PostInfo.scss';
+import { Posts } from '../../types/Posts';
+import { CommentList } from '../CommentList';
+import { UserInfo } from '../UserInfo';
 
-export const PostInfo: React.FC = () => (
-  <>Put the post here</>
-);
+type Props = {
+  post: Posts,
+};
+
+export const PostInfo: React.FC<Props> = ({
+  post: {
+    title,
+    user,
+    body,
+    comments,
+  },
+}) => {
+  const renderComments = () => {
+    if (!comments?.length) {
+      return (<b data-cy="NoCommentsMessage">No comments yet</b>);
+    }
+
+    return (<CommentList comments={comments} />);
```

Comment:
```suggestion
    return <CommentList comments={comments} />;
```
Try in that way make your returns.